config/locales/en.yml

```diff
@@ -8,4 +8,5 @@ en:
 
   button:
     next: "Next"
-    back: "Back"
\ No newline at end of file
+    back: "Back"
+    submit: "Submit Order"
```

Comment:
Good that it is added in translation file!

Reply:
ok